src/vs/editor/common/config/editorOptions.ts

```diff
@@ -2523,19 +2523,21 @@ export interface IEditorExperimentalOptions {
 		 * Enable the sticky scroll
 		 */
 		enabled?: boolean;
+		numberLines?: number;
```

Comment:
How about we rename this to something like `maxLineCount`? `numberLines` could be interpreted as always having a certain number of lines.